compiler/src/dotty/tools/dotc/core/tasty/TastyUnpickler.scala

```diff
@@ -49,7 +49,7 @@ class TastyUnpickler(reader: TastyReader) {
       case UTF8 =>
         goto(end)
         termName(bytes, start.index, length)
-      case QUALIFIED | FLATTENED | EXPANDED | EXPANDPREFIX =>
+      case QUALIFIED | FLATTENED | EXPANDED | EXPANDPREFIX | TRAITSETTER =>
```

Comment:
I had not realized TRAITSETTER could reach the pickler. I thought they were added after pickling. What's the scenario where this came up?